jasmine/src/roll.js

```diff
@@ -0,0 +1,19 @@
+function Roll(game, frame){
+  this.game = game;
+  this.frameScore = null;
```

Comment:
By convention, these values should have underscores and we should try not to access them from outside this class, i.e. 

```
this._isStrike
this._isSpare
```

Also, do isStrike and isSpare need to be class variables? You only use them in the roll function, so I would suggest that you could get rid of them and your roll function could be:

```
Roll.prototype.roll = function(score1, score2) {
  var isStrike = (score1 == 10);
  var isSpare = !isStrike && (score2 + score1 == 10)
  this.addScore(score1, score2, isStrike, isSpare);
};
```